src/osha/oira/client/browser/client.py

```diff
@@ -31,8 +36,12 @@ def results(self):
             portal_type="euphorie.clientcountry",
             sort_on="sortable_title",
         )
+        client_path = "/".join(self.context.getPhysicalPath())
         results.extend(
-            [{"id": brain.getPath(), "text": brain.Title} for brain in brains]
+            [
+                {"id": path.relpath(brain.getPath(), client_path), "text": brain.Title}
```

Comment:
It gives us a path relative to the client, like in other tables, e.g. `de/construction/construction` instead of `/Plone2/client/de/construction/construction`.

I would risk this breaking on Windows, but if you think it's a problem I'll change it.